### DISCUSSION TITLE
Indicate where to buy HARP

### DISCUSSION THREAD

**Discussion by pipcoen** [harp-tech/protocol#96] [Open-ended]
Hi all. I just wanted to raise the issue that there is no indication on https://harp-tech.org/ as to where you can actually obtain these products. I do understand that these are being made and sold by independent companies, and that there is value in remaining separate from that. 

However, it's difficult for me to recommend HARP without these instructions because people don't know where to get them! I think a guide for how to buy the products (with a disclaimer etc.) would be great. Apologies if this already exists somewhere, and I've missed it.

**Comment by filcarv** [comment]
Hi.
Open Ephys is currently actively manufacturing and distributing Harp devices worldwide. They will become available in the store during Q3 of 2024.
Meanwhile, feel free to reach out info[at]oeps.tech for more information and ordering.

**Reply by pipcoen** [reply]
Ah, that's great to know they will be in the store by Q3. I'll pass it on to my colleague. Thanks!
